doc/contributing/PROCESS.md

```diff
@@ -174,6 +174,92 @@ To make a new OpenAssetIO release, follow this procedure.
+they also have their own CI suites to validate any changes made to them,
+using a released OpenAssetIO version, taken from PyPI.
+
+### Breaking Change Mitigation Process
+
+On occasion, it is necessary for OpenAssetIO to change its interfaces in
+such a way that cause the downstream projects to become incompatible
+against the bleeding edge OpenAssetIO. You will know when this happens
+as one of the tests in
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml)
+will begin to fail.
+
+The downstream projects are pinned to a specific version of OpenAssetIO
+on PyPI for their own test suites, so these will continue to function
+even through a breaking change on OpenAssetIO main. However, the
+OpenAssetIO integrations checks will being to fail, with no clear way to
+remedy the situation before an OpenAssetIO release, as any changes to
+the downstream project to support bleeding edge will likely cause
+incompatibility with the published OpenAssetIO release the project is
+consuming from PyPI.
+
+In order to remedy this, when making a breaking OpenAssetIO change that
+causes a failing integrations check, the following procedure may be
+implemented to avoid long term loss of coverage.
+
+- Consult on whether the change must necessarily be breaking, if
+reasonably possible, attempt to formulate another approach.
+
+- Evaluate the scope of breakage in the dependent project(s), and make
+an issue in those projects to fix compatibility with your change.
+
+- Comment out the failing integrations check in
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml).
+Leave a note explaining the context, with links to the relevant issues.
+
+- Review and merge your change to OpenAssetIO as normal, call out the
+removed integrations check in the PR.
+
+- As soon as possible, plan to work on the issues to return
+compatibility to the dependent repos.
+
+- To accomplish this, branch the dependent repo in question from main,
+call your new branch `openassetio/X.Y.Z`
+
+  > **Warning**
+  >
+  > The specific branch name formatting is important, as CI is setup to
+  > source wheels built from the tip of `openassetio:main` when run on
+  > these branches
+
+- Fix the project to function with the new interface. Review and Merge
+to the `openassetio/X.Y.Z` branch.
+
+- Re-enable the disabled integration tests in OpenAssetIO by re-targeting
+the checkout in
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml)
+to point to this new branch. It should now pass.
+
+- When OpenAssetIO has its next release, fold these branches back into
+  the main branches and delete them.
+
+- Re-target
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml)
+back to normal, checking out the `main` branch of the dependent repos.
```

Comment:
[consider] I know it should be pretty obvious, but to be safe we shouldn't delete the staging branch until there's no chance of it being used.

```suggestion
- When OpenAssetIO has its next release, fold these branches back into
  the main branches.

- Re-target
[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml)
back to normal, checking out the `main` branch of the dependent repos.

- Delete the `openassetio/X.Y.Z` staging branch.
```

Reply:
Really good call imo, [9d04a58](https://github.com/OpenAssetIO/OpenAssetIO/pull/772/commits/9d04a58afb9f23a32f2d16986eecc982a1949b68)